docs/www/cr-additional-config.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
can the file be renamed `kubernetes-additional-config.md`

Reply:
the `cr` url doesn't make sense for ppl

Reply:
Sure

Reply:
Sure. This article is not referenced anywhere but I can add a redirect for it anyway.

> @bmansheim - if we rename, we need to start to think about permanent urls and redirects that @yougotashovel built for [www](http://www). not sure if this falls into this but the old docker urls for example broke when we move to the new titles, etc.
> 
> so when we change the name of the file, we also change the url it goes to.